lua/octo/menu.lua

```diff
@@ -731,4 +731,60 @@ function M.changed_files()
   )
 end
 
+---
+-- SEARCH
+---
+
+function M.issue_search(repo, opts)
+  opts = opts or {}
+
+  if not repo or repo == vim.NIL then
+    repo = util.get_remote_name()
+  end
+  if not repo then
+    api.nvim_err_writeln("Cannot find repo")
+    return
+  end
+
+  pickers.new(
+    opts,
+    {
+      prompt_title = "Issue Search",
+      finder = function(prompt, process_result, process_complete)
+        if not prompt or prompt == "" then
+          return nil
+        end
+        prompt = util.escape_chars(prompt)
+        local filter = prompt
+        local query = format(graphql.search_issues_query, repo, filter)
+        --print(query)
+        gh.run(
+          {
+            args = {"api", "graphql", "-f", format("query=%s", query)},
+            cb = function(output, stderr)
+              if stderr and not util.is_blank(stderr) then
+                api.nvim_err_writeln(stderr)
+              elseif output then
+                local resp = json.parse(output)
+                --print(#resp.data.search.nodes)
+                for _, issue in ipairs(resp.data.search.nodes) do
+                  process_result(gen_from_issue(4)(issue))
+                end
+                process_complete()
+              end
+            end
+          }
```

Comment:
It seems like there is a debouncing problem and the finder ends up accumulating the results from previous invocations (partial prompts)

@tjdevries @tami5 @conni2461 do you see anything wrong in the custom finder? any ways to improve it and discard results from partial prompts?